pdf_report/pdf_report.py

```diff
@@ -0,0 +1,23 @@
+"""Main script for generating the daily report PDF and mailing it to all subscribers."""
+
+import logging
+from generate_pdf import create_pdf
+from mailer import send_all_emails
+
+
+def main(event, context):  # pylint: disable=unused-argument
+    """Main function to execute the:
+        1. Generation of the daily report PDFs
+        2. Sending of PDFs to all subscribers
+    """
+
+    logging.basicConfig(
+        level=logging.WARNING, format="%(asctime)s - %(levelname)s - %(message)s"
+    )
+
```

Comment:
This was already called inside the function

Reply:
Yes, but I _think_  that because it is a different file, the logger has to be configured again?

Reply:
Actually... you might be right...

Reply:
I think so as well